subsys/net/lib/sockets/sockets.c

```diff
@@ -531,7 +531,11 @@ static inline ssize_t zsock_recv_dgram(struct net_context *ctx,
 		recv_len = max_len;
 	}
 
-	net_frag_linearize(buf, recv_len, pkt, header_len, recv_len);
+	/* Length passed as arguments are all based on packet data size
+	 * and output buffer size, so return value is invariantly == recv_len,
+	 * and we just ignore it.
+	 */
+	(void)net_frag_linearize(buf, recv_len, pkt, header_len, recv_len);
```

Comment:
Are you sure you don't want to handle -ENOMEM errors here?

Reply:
Hmm..  looking at code:
- You won't get the first -ENOMEM because you're passing recv_len for both length params
- You probably won't get second -ENOMEM because recv_len is set via net_pkt_appdatalen() which is usually set via net_buf_frags_len().  This function iterates through the buf frags in much the same was as linearize.

TL;DR: Pretty small chance of an error here.

Reply:
> Are you sure you don't want to handle -ENOMEM errors here?

There won't be any -ENOMEM's, sorry if I didn't cc: you on https://github.com/zephyrproject-rtos/zephyr/pull/12230 .